app/build.gradle

```diff
@@ -43,7 +43,9 @@ android {
             buildConfigField "String", "MAPBOX_KEY", '"'+MAPBOX_KEY+'"'
         }
         debug {
-            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
+//            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
```

Comment:
Wrong file committed

Reply:
oh right, but I'm gonna split this PR now, one for loadEvent() function, the other for sharedElementTransition